cypress/component/integration/Dropdown.cy.tsx

```diff
@@ -0,0 +1,289 @@
+        compareSnapshot(cy, 'default-xs-dropdown');
+      });
+    });
+
+    describe('default md dropdown', () => {
+      beforeEach(() => {
+        cy.mount(
+          <div className="theme-dark">
+            <Dropdown
+              size={'md'}
+              options={[
+                {
+                  icon: (
+                    <MassaToken
+                      size={32}
+                      className="bg-c-default rounded-full"
+                    />
+                  ),
+                  item: 'account 1 name',
+                  onClick: () => console.log('option 1'),
+                },
+                {
+                  icon: <MassaLogo size={32} />,
+                  item: 'account 2 name with a biiiiig content',
+                },
+                { icon: <MassaLogo size={32} />, item: 'account 3 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 4 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 5 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 6 name' },
+              ]}
+            />
+            ,
+          </div>,
+        );
+      });
+
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-md-dropdown');
+      });
+    });
+
+    describe('default dropdown flow', () => {
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+
+        it('should toggle when clicked', () => {
+          cy.get('[data-testid="dropdown"]').should('exist');
+
+          for (let i = 1; i <= 6; i++) {
+            cy.get(`[data-testid="dropdown-option-${i}"]`).should('exist');
+          }
+
+          cy.get('[data-testid="dropdown-option-1"]').should('be.visible');
+          for (let i = 2; i <= 6; i++) {
+            cy.get(`[data-testid="dropdown-option-${i}"]`).should(
+              'not.be.visible',
+            );
+          }
+        });
+
+        // open
+        cy.get('[data-testid="dropdown-button"]').should('exist').click();
+
+        for (let i = 1; i <= 6; i++) {
+          cy.get(`[data-testid="dropdown-option-${i}"]`).should('be.visible');
+        }
+
+        // close
+        cy.get('[data-testid="dropdown-button"]').should('exist').click();
+
+        cy.get('[data-testid="dropdown-option-1"]').should('be.visible');
+        for (let i = 2; i <= 6; i++) {
+          cy.get(`[data-testid="dropdown-option-${i}"]`).should(
+            'not.be.visible',
+          );
+        }
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-dropdown-flow');
+      });
+    });
+
+    describe('select option dropdown flow', () => {
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+        it('should open and close when clicked', () => {
```

Comment:
... same here. This is nested under render. 

Secondly you already tested open/close before. So, only once is enough.

Reply:
same